data/chameleoncloud/sites/uc/clusters/chameleon/nodes/16035a48-fc53-4b5f-87fc-f911569545da.json

```diff
@@ -5,91 +5,93 @@
     "version": "2.10.2"
   },
   "chassis": {
-    "manufacturer": "Dell Inc.",
+    "manufacturer": "Dell",
     "name": "PowerEdge R740",
     "serial": "76G4KD3"
   },
+  "gpu": {
+    "gpu": false
+  },
   "main_memory": {
     "humanized_ram_size": "192 GiB",
-    "ram_size": 192000000000
+    "ram_size": 206158430208
   },
   "monitoring": {
     "wattmeter": false
   },
   "network_adapters": [
     {
-      "device": "NIC.Integrated.1-1",
+      "device": "eno1np0",
+      "driver": "mlx5_core",
       "enabled": true,
       "interface": "Ethernet",
       "mac": "b8:ce:f6:43:21:6e",
-      "management": false,
-      "model": "MLNX 25GbE 2P ConnectX4LX RNDC",
+      "model": "MT27710 Family [ConnectX-4 Lx]",
       "rate": 25000000000,
-      "vendor": "Mellanox Technologies"
+      "vendor": "Mellanox"
     },
     {
-      "device": "NIC.Integrated.1-2",
+      "device": "eno2np1",
+      "driver": "mlx5_core",
       "enabled": true,
       "interface": "Ethernet",
       "mac": "b8:ce:f6:43:21:6f",
-      "management": false,
-      "model": "MLNX 25GbE 2P ConnectX4LX RNDC",
+      "model": "MT27710 Family [ConnectX-4 Lx]",
       "rate": 25000000000,
-      "vendor": "Mellanox Technologies"
+      "vendor": "Mellanox"
     },
     {
-      "device": "NIC.Slot.7-1",
+      "device": "ens7f0",
+      "driver": "tg3",
       "enabled": false,
       "interface": "Ethernet",
       "mac": "e4:3d:1a:2d:db:62",
-      "management": false,
-      "model": "BRCM GbE 2P 5720-t Adapter",
-      "vendor": "Broadcom Inc. and subsidiaries"
+      "model": "NetXtreme BCM5720 Gigabit Ethernet PCIe",
+      "rate": 1000000000,
+      "vendor": "Broadcom"
     },
     {
-      "device": "NIC.Slot.7-2",
+      "device": "ens7f1",
+      "driver": "tg3",
       "enabled": false,
       "interface": "Ethernet",
       "mac": "e4:3d:1a:2d:db:63",
-      "management": false,
-      "model": "BRCM GbE 2P 5720-t Adapter",
-      "vendor": "Broadcom Inc. and subsidiaries"
+      "model": "NetXtreme BCM5720 Gigabit Ethernet PCIe",
+      "rate": 1000000000,
+      "vendor": "Broadcom"
     }
   ],
   "node_name": "P3-CPU-004",
   "node_type": "compute_cascadelake_r",
   "placement": {
     "node": "12",
-    "rack": "BI-41"
+    "rack": "5"
   },
   "processor": {
-    "cache_l1": 1536000,
-    "cache_l1d": null,
-    "cache_l1i": null,
-    "cache_l2": 24576000,
-    "cache_l3": 36608000,
+    "cache_l1d": 32768,
+    "cache_l1i": 32768,
+    "cache_l2": 1048576,
+    "cache_l3": 37486592,
     "clock_speed": 2400000000,
-    "instruction_set": "x86-64",
-    "model": "Intel Xeon",
-    "other_description": "Intel(R) Xeon(R) Gold 6240R CPU @ 2.40GHz",
-    "vendor": "Intel",
-    "version": "Model 85 Stepping 7"
+    "instruction_set": "x86_64",
+    "model": "Intel(R) Xeon(R) Gold 6240R CPU @ 2.40GHz",
+    "vendor": "Intel"
   },
   "storage_devices": [
     {
-      "device": "Disk.Bay.0:Enclosure.Internal.0-1:NonRAID.Slot.6-1",
+      "device": "sda",
```

Comment:
similar to network cards, it would be nice if we could also grab the physical connection info somehow.